deps/rabbit/src/rabbit_db.erl

```diff
@@ -98,7 +122,8 @@ force_reset_using_mnesia() ->
 %% state, like if critical members are MIA.
 
 force_load_on_next_boot() ->
-    force_load_on_next_boot_using_mnesia().
+    rabbit_db:run(
+      #{mnesia => fun force_load_on_next_boot_using_mnesia/0}).
 
 force_load_on_next_boot_using_mnesia() ->
     ?LOG_DEBUG(
```

Comment:
It might not belong to this PR, but the below message is incorrect. Force load doesn't reset the node, it forces the node to start if it was not the last node going down.

Reply:
Indeed, fixed!